### ISSUE TITLE
Unix support

### ISSUE THREAD

**Issue by pug-pelle-p** [poxet/tharga-console#16]
Right now if you start the application on a unix based system, it would crash due to it throwing an PlatformNotSupportedException because System.Console.MoveBufferArea (Only used in ConsoleManager.MoveBufferArea() ) is not supported on unix systems.

If I put a try-catch around it and swallow the exception, the application works, but the text won't show until I start typing something. Not sure how to fix this, any ideas?

**Comment by poxet** [comment]
If the user is typeing something, and the application writes a line of text, the buffer is moved down, so that the input is always at the bottom.

The PlatformNotSupportedException for MoveBufferArea seems to have been for several years, so a solution does not seem to be likley in the near future.

I cannot figure out another way of solving this. Perhaps having the program output be written on lines under the input, but it might be very strange to use. And allot to rewrite.

**Comment by poxet** [comment]
Would a solution be to "pause" the output, if the user is typing something? Perhaps an indication on the title bar, that there are pending lines to be written, once the user is done typing.

**Comment by pug-pelle-p** [comment]
Hmm okay, so do I understand you correctly that the usage of MoveBufferArea is to support the use of an application sending messages after a command is completed or from some background thread?

**Comment by poxet** [comment]
Yes. If the user starts typing something, and the background process outputs data, then the input buffer is moved down, to make room for the output. If the output is suspended until the user is done typing, then the input would be visible.

**Comment by poxet** [comment]
Closing as not planned. This has been open for years without further interest or a PR. Happy to reconsider if someone wants to drive it.
